KeychainExample/App.js

```diff
@@ -87,6 +87,9 @@ export default class KeychainExample extends Component {
   async load() {
     try {
       const options = {
+        accessControl: this.state.accessControl,
```

Comment:
These must be passed to `getGenericPassword` call to force strong biometrics.

Reply:
As we discussed it would be preferable if this could be inferred like it works on iOS instead.

Reply:
Inferred this from the storage type and removed the params.